t/api/05-machines.t

```diff
@@ -83,8 +83,11 @@ is_deeply(
 ) || diag explain $get->tx->res->json;
 
 
-my $res = $t->post_ok('/api/v1/machines', form => {name => "testmachine"})->status_is(400);    # missing backend
-$res = $t->post_ok('/api/v1/machines', form => {backend => "kde/usb"})->status_is(400);        #missing name
+my $res = $t->post_ok('/api/v1/machines', form => {name => "testmachine"})->status_is(400)
+  ->json_is('/error', 'Missing parameter: backend');    # missing backend
+$t->post_ok('/api/v1/machines', form => {backend => "kde/usb"})->status_is(400)
+  ->json_is('/error', 'Missing parameter: name');       #missing name
```

Comment:
The comments here don't really add new information, it's already in the error message now.